data/movements/scripts/walkback.lua

```diff
@@ -1,6 +1,6 @@
 function onStepIn(creature, item, position, fromPosition)
 	if item.uid > 0 and item.uid <= 65535 then
-		creature:teleportTo(fromPosition, false)
+		return false
 	end
 	return true
```

Comment:
Nope, see https://github.com/otland/forgottenserver/blob/master/data/lib/compat/compat.lua#L74 and https://github.com/otland/forgottenserver/blob/master/src/luascript.cpp#L5908, actually I think item.uid == 0 is the useless one.